library/src/main/java/com/tolstykh/textviewrichdrawable/DrawableEnriched.java

```diff
@@ -17,4 +19,13 @@
      * @return the width in pixels or {@value UNDEFINED} if undefined.
      */
     int getCompoundDrawableWidth();
+
+    void setDrawableStartVectorId(@DrawableRes int id);
+
+    void setDrawableEndVectorId(@DrawableRes int id);
+
+    void setDrawablTopVectorId(@DrawableRes int id);
```

Comment:
There is a typo in the method name: setDrawablTopVectorId -> setDrawableTopVectorId ('e' is missing in the end of drawable)